tests/test_kubectl.py

```diff
@@ -0,0 +1,33 @@
+
+import pytest
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerLauncher
+from pytest_container.runtime import OciRuntimeBase
+
+from bci_tester.data import KUBECTL_CONTAINERS
+
+
+@pytest.mark.parametrize("ctr_image", KUBECTL_CONTAINERS)
+def test_kubectl_binary(
+    container_runtime: OciRuntimeBase,
+    pytestconfig: pytest.Config,
+    ctr_image: DerivedContainer,
+) -> None:
+    kubectl_ctr = DerivedContainer(
+        base=ctr_image,
+        custom_entry_point="/bin/bash",
+    )
+
+    with ContainerLauncher.from_pytestconfig(
+        kubectl_ctr, container_runtime, pytestconfig
+    ) as kubectl_launcher:
+        kubectl_launcher.launch_container()
+        kubectl_con = kubectl_launcher.container_data.connection
+        output = kubectl_con.check_output(
+            "kubectl create deployment nginx --image=nginx --dry-run=client -o yaml"
+        ).strip()
+        assert len(output) == 383
+        assert (
+            output
+            == "apiVersion: apps/v1\nkind: Deployment\nmetadata:\n  creationTimestamp: null\n  labels:\n    app: nginx\n  name: nginx\nspec:\n  replicas: 1\n  selector:\n    matchLabels:\n      app: nginx\n  strategy: {}\n  template:\n    metadata:\n      creationTimestamp: null\n      labels:\n        app: nginx\n    spec:\n      containers:\n      - image: nginx\n        name: nginx\n        resources: {}\nstatus: {}"
```

Comment:
Please use multi-line strings, if possible:
```suggestion
            == """apiVersion: apps/v1
kind: Deployment
metadata:
  creationTimestamp: null
  labels:
    app: nginx
  name: nginx
spec:
  replicas: 1
  selector:
    matchLabels:
      app: nginx
  strategy: {}
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: nginx
    spec:
      containers:
      - image: nginx
        name: nginx
        resources: {}
status: {}"""
```